federator/src/contracts/IFederationV4.ts

```diff
@@ -6,9 +6,9 @@ interface TransactionIdParams {
   sender: string;
   receiver: string;
   amount: number;
```

Comment:
If sending large amounts does it hadnle it correctly

Reply:
Donet at https://github.com/rsksmart/tokenbridge/pull/286/commits/e1eaee1f5e99eafc950a5a90b9c2d3dfd3d9ac9f

Reply:
Thanks :)